andino_description/urdf/andino.urdf.xacro

```diff
@@ -8,21 +8,23 @@
   <xacro:include filename="$(find ${package_name})/urdf/include/common_sensors.urdf.xacro" />
   <xacro:include filename="$(find ${package_name})/urdf/include/andino_caster_macro.urdf.xacro" />
 
-  <xacro:property name="caster_wheel_yaml" value="$(find ${package_name})/config/${robot_name}/caster_wheel.yaml" />
+  <xacro:arg name="config_yaml_path" default="$(find ${package_name})/config/${robot_name}" />
```

Comment:
nit: I would change the name to `yaml_config_dir`  given that we are actually passing the path to the directory where the yaml files are expected to be located.